californium-core/src/main/java/org/eclipse/californium/core/network/Exchange.java

```diff
@@ -129,13 +135,12 @@
 	// handle to cancel retransmission
 	private ScheduledFuture<?> retransmissionHandle = null;
 
-	// handle to extend blockwise status lifetime
-	private ScheduledFuture<?> blockCleanupHandle = null;
-
 	// If the request was sent with a block1 option the response has to send its
 	// first block piggy-backed with the Block1 option of the last request block
 	private BlockOption block1ToAck;
 
+	private Integer observeNo;
```

Comment:
Why is this field named "observe..." but used only in "notification...()" methods?
Doc comment missing.